client/src/components/App.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'React';
+import Header from './Header.jsx';
+import Modal from './Modal.jsx';
+import Navbar from './Navbar.jsx';
+import Description from './Description.jsx';
+import Footer from './Footer.jsx';
+
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { 
+      restaurant: null,
+      currentImage: 2,
+      images: ['https://i.imgur.com/i2FffSR.jpg', 'https://i.imgur.com/rd9yHKH.jpg', 'https://i.imgur.com/ocPGEcz.jpg', 'https://i.imgur.com/noBxGLl.jpg', 'https://i.imgur.com/yajIWRO.jpg']
+    }
+  }
+
+  componentDidMount(){
+    this.getData()
+  }
+
+  getData() {
+    $.ajax({
+      url: '/restaurant/overview',
+      method: 'GET',
+      dataType: 'json',
+      success: (data) => {
+      console.log('success', data)
+      this.setState({
+        restaurant: data[0]
+      })
+    },
+      error: () => {console.log('error')}
+    })    
+  }
+
+  handleModal(e) {
+    console.log('Clicked', e.target.id)
+    if(e.target.id !== "bookmark"){
+    document.getElementById('myModal').style.display = "block";
```

Comment:
Grabbing elements from the document like this is a React Anti-pattern! Try to refactor this eventually to do this without accessing the DOM directly